devito/function.py

```diff
@@ -1006,37 +985,36 @@ def inject(self, field, expr, offset=0, u_t=None, p_t=None):
                     field.subs(vsub) + expr.subs(subs).subs(vsub) * b.subs(subs))
                 for b, vsub in zip(self.coefficients, idx_subs)]
 
-    def argument_defaults(self, alias=None):
+    def _arg_defaults(self, alias=None):
         """
         Returns a map of default argument values defined by this symbol.
 
         :param alias: (Optional) name under which to store values.
         """
         key = alias or self
-        args = super(SparseFunction, self).argument_defaults(alias=alias)
-        args.update(self.coordinates.argument_defaults(alias=key.coordinates))
 
+        args = super(SparseFunction, self)._arg_defaults(alias=alias)
+        args.update(self.coordinates._arg_defaults(alias=key.coordinates))
         return args
 
-    def argument_values(self, alias=None, **kwargs):
+    def _arg_values(self, alias=None, **kwargs):
```

Comment:
This function should not be necessary, but I can just redo my branch once this is merged,.